cache/server.go

```diff
@@ -116,9 +116,13 @@ type clientProtocolServer struct {
 var _ common.StarterShutdowner = (*clientProtocolServer)(nil)
 
 func newClientProtocolServer(l *logrus.Logger, c *Cache, db *sql.DB, conf *ConfigFile, r *ledgerclient.Replicator) (*clientProtocolServer, error) {
-	grpcServer := common.NewDBGRPCServer(db)
+	// Cache security is currently delivered solely via the cachecash crypto
+	// guarantees. TLS certs will be added in future.
+	grpcServer := common.NewDBGRPCServer(true, db)
 	ccmsg.RegisterClientCacheServer(grpcServer, &grpcClientCacheServer{cache: c})
 	ccmsg.RegisterPublisherCacheServer(grpcServer, &grpcPublisherCacheServer{cache: c})
+	// Enable to observe gRPC latency
+	// grpc_prometheus.EnableHandlingTimeHistogram()
```

Comment:
Why are we leaving in commented code?

Reply:
It is only required for performance analysis, so benchmarking. Therefore, I put this comment until adding a configuration parameter in LogPiped to enable/disable this feature. A person who is not an expert on Prometheus may spend so much time to figure out that.